jac-uk/admin
# WIP: #1347 Enable application content to be assigned to exercise state

## What's included?
Enables application content to be assigned to exercise states. See #1347 for details.
Also introduces basic drag & drop components

## Who should test?
✅ Product owner

## How to test?
In Admin, find or configure a draft exercise.
Navigate to the new 'Application process' page.
Click to 'Update application process'.
Drag and drop application 'parts' to the desired steps of the application process.
Click 'Continue' and notice that the changes have been saved.

Edit the exercise configuration so that the relevant application 'parts' change. For example make changes to the 'Assessment options'. 
Navigate to 'Application process' and notice that the new 'parts' have been added to 'Registration'
Try 'Update application process' again (as above).

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional information
https://user-images.githubusercontent.com/8524401/121855469-4744bc80-cceb-11eb-927f-36b2939dc95e.mov

https://user-images.githubusercontent.com/8524401/121855724-968aed00-cceb-11eb-92b3-f13b458acf96.mov


---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
